docs/docs/querying-with-graphql.md

```diff
@@ -19,6 +21,10 @@ components](/docs/building-with-components/) to declare what data they and their
 sub-components need. Then, Gatsby makes that data available in
 the browser when needed by your components.
 
+Data from any number of sources is made queryable in one unified layer:
```

Comment:
Because the interactive piece includes other aspects of Gatsby while focusing on the data layer by default, it would help to mention somehow why the other sections are present. How does data relate to the rest of it?

Here's one attempt, let me know what you think: "Data from any number of sources is made queryable in one unified layer, a key part of Gatsby the building process:"

Reply:
Good point, I like how that's phrased. I think the paragraph above this one helps give some context as well since it explains that data is made available to components in the browser and hopefully the little examples can help that sink in.